Groups changes (#145)
music-assistant/music-assistant.github.io: docs/faq/groups.md

    
## Temporary Sync Group

> Temporary Sync Group players are configured via the [Player List](ui.md#player-list) or HA action. If multiple HA players are available to MA and those players can be grouped in HA then that functionality will also be available via this method. This type of group provides a very flexible way to create and destroy Sync Groups as the players can be added or removed at any time. 

Author: I think this was in relation to how for a period (I am assuming it has been fixed) HA players weren’t able to be synced even though the underlying HA integration supported syncing…. I haven’t got any players that support this so I don’t know how it looks in the UI?

> Temporary Sync Group players are configured via the [Player List](ui.md#player-list) or HA action. If multiple HA players are available to MA and those players can be grouped in HA then that functionality will also be available via this method. This type of group provides a very flexible way to create and destroy Sync Groups as the players can be added or removed at any time. 

Reviewer: it will look the same as for other players that can be synced

> Temporary Sync Group players are configured via the [Player List](ui.md#player-list) or HA action. If multiple HA players are available to MA and those players can be grouped in HA then that functionality will also be available via this method. This type of group provides a very flexible way to create and destroy Sync Groups as the players can be added or removed at any time. 

Reviewer: ```suggestion
Temporary Sync Group players are configured via the [Player List](ui.md#player-list) or HA action. This type of group provides a very flexible way to create and destroy Sync Groups as the players can be added or removed at any time. 
```

## Sync Groups

These are permanent group players that are configured via the MA settings. These should be used when the make up of the group will rarely change and it is desired that the players will typically all continuously play music. A player cannot be removed from the group if the group is playing although it can be muted (or volume reduced to 0). If there is a need to add or remove additional players then enable the dynamic members option or switch to using a manual sync group. At power on/off of the syncgroup the original set of members will be restored.

![image](../assets/screenshots/syncgroup.png)

> After selecting ADD GROUP PLAYER as shown above, the group configuration screen will appear where the group type must be selected. It is recommended that a custom name be used as the automatically generated name is unique but not memorable. The order that the players are selected when the group is created will be the order that the players are shown when viewing the group in the [Player List](../ui.md#player-list).

Reviewer: ```suggestion
After selecting ADD GROUP PLAYER as shown above, the group configuration screen will appear where the group type must be selected. You need to specify a name for the group player. The order that the players are selected when the group is created will be the order that the players are shown when viewing the group in the [Player List](../ui.md#player-list).
```

!!! tip
    Remember Chromecast devices must be grouped in the Google Home app for perfect sync

### Features

- The native player types shown (i.e. not the Universal Group option) support perfect sync
- In addition to playing in sync the group will hold the queue regardless of which player(s) in the group are powered off
  
### Known Issues / Notes

- As mentioned above the group player holds the queue rather than any individual player. However, if the group leader is lost then playback will stop but the queue can be restarted
- The group will not power on if one of the child players is synced to another group
- If, in the group settings, the player friendly name has been replaced by an unusual ID then likely the player was seen by the MA logic as unavailable. Check the individual player status

> - When adding a player to an already playing group, Airplay, Snapcast and Sonos will keep playing but all other types will pause for a short period

Reviewer: ```suggestion
- When adding a player to an already playing group, some player types will pause for a very brief period to resync.
```

Author: The reason I added a list is because people ask if the pause they experienced is normal or a bug? This answers the question for them?

## Universal Groups

Music Assistant has support for grouping dissimilar playback devices. Configuration is done in the same manner as described in the Sync Groups section with UNIVERSAL selected as the Group Type. These groups also support the dynamic member option.

### Features

> - All devices can be grouped but not all will play in sync. See the individual player providers page for any specific limitations

Reviewer: ```suggestion
- All devices can be grouped and will play the same audio but will not play the audio in sync. Use this playertype only for players that are not in close range to eachother.
- See the individual player providers page for any specific limitations
```
